app/pods/agenda/agendaitems/agendaitem/documents/controller.js

```diff
@@ -180,6 +183,22 @@ export default class DocumentsAgendaitemsAgendaController extends Controller {
     }
   }
 
+  @action
+  async openPublicationModal() {
+    this.publicationModalDataPromise = this.store.findRecord('agendaitem', this.agendaitem.id, {
```

Comment:
Ik wilde hier zorgen dat de gereferencete models geladen zijn: kwestie van DDAU en voor het aantal calls te minimaliseren, maar misschien is hiervoor een betere methode.

Reply:
Dit is nu gewijzigd.